domain/schema/model/sql/0035-cleanup.PATCH.sql

```diff
@@ -1,2 +1,4 @@
+-- controller-model removal needs to be handled differently to
+-- regular model removal
 INSERT INTO removal_type VALUES
 (15, 'controller-model');
```

Comment:
We haven't released 4.0.1 yet, so this change is valid. I've included to demo the test still passes.